glue/clients/scatter_client.py

```diff
@@ -111,12 +114,26 @@ def _set_limits(self, *args):
         self.axes.set_ylim(ylim)
         if xlim != xold or ylim != yold:
             self._redraw()
+            self._update_axis_labels()
 
     def plottable_attributes(self, layer, show_hidden=False):
         data = layer.data
         comp = data.components if show_hidden else data.visible_components
         return [c for c in comp if
-                data.get_component(c).numeric]
+                data.get_component(c).numeric or data.get_component(c).datetime]
+
+    def groupable_attributes(self, layer, show_hidden=False):
+        data = layer.data
+        if not data.find_component_id('None'):
+            l = data._shape[0]
+            none_comp = Component(np.array(range(0, l)), units='None')
+            data.add_component(none_comp, 'None', hidden=False)
```

Comment:
I don't like the idea of the scatter plot adding data, since it doesn't make sense in the context of other data viewers. What is the intention of the None component? Is there another way to get this to work?